contracts/plugins/assets/curve/cvx/vendor/ConvexStakingWrapper.sol

```diff
@@ -38,9 +39,18 @@ interface IConvexDeposits {
     ) external;
 }
 
+interface IRewardHook {
+    function onRewardClaim() external;
+}
+
+interface ITokenWrapper {
+    function token() external view returns (address);
+}
+
 // if used as collateral some modifications will be needed to fit the specific platform
 
-// Based on audited contracts: https://github.com/convex-eth/platform/blob/main/contracts/contracts/wrappers/CvxCrvStakingWrapper.sol
+// Based on audited contracts: https://github.com/convex-eth/platform/blob/main/contracts/contracts/wrappers/ConvexStakingWrapper.sol
+// Commit hash: 933ace3
```

Comment:
Should use the full commit hash to be able to reproduce it, like so: https://github.com/convex-eth/platform/blob/933ace34d896e6684345c6795bf33d4089fbd8f6/contracts/contracts/wrappers/ConvexStakingWrapper.sol